pmemcheck/pmc_main.c

```diff
@@ -98,6 +98,12 @@ static struct pmem_ops {
     /** Holds the number of registered erroneous flush events. */
     UWord flush_errors_reg;
 
+    /** Holds unnecessary flush error events. */
+    struct pmem_st **flush_misses;
```

Comment:
This variable name is quite misleading - are these *unnecessary* or *missing* flushes?